routing/routing_session.cpp

```diff
@@ -523,6 +523,26 @@ void RoutingSession::EmitCloseRoutingEvent() const
       alohalytics::Location::FromLatLon(lastGoodPoint.lat, lastGoodPoint.lon));
 }
 
+bool RoutingSession::HasRouteAltitudeImpl() const
+{
+  return !m_route.GetAltitudes().empty();
+}
+
+bool RoutingSession::HasRouteAltitude() const
+{
+  threads::MutexGuard guard(m_routeSessionMutex);
+  return HasRouteAltitudeImpl();
+}
+
+bool RoutingSession::CopyRouteAltitudes(feature::TAltitudes & routeAltitudes) const
```

Comment:
I don't like the method looks like a light weight method. If I call it GetRouteAltitudes() users could consider it as a quick getter but it copies all the content of vector with significant length.